dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/DockerfileGitHubUtil.java

```diff
@@ -177,10 +177,11 @@ protected String getOrgNameWithMaximumHits(PagedSearchIterable<GHContent> files)
         Optional<List<PagedSearchIterable<GHContent>>> contentsForOrgWithMaximumHits;
         contentsForOrgWithMaximumHits = findFilesWithImage(image, orgsToInclude, gitApiSearchLimit);
 
-        orgsToExclude.put(orgWithMaximumHits, false);
-        log.info("Running search by excluding the orgs {}.", orgsToExclude.keySet().toString());
+        final Map<String, Boolean> orgsToExcludeFromSearch = new HashMap(orgsToExclude);
```

Comment:
You'll want to use `new HashMap<>(orgsToExclude)` otherwise you'll get type warnings during compilation.